students/saleta_sebastian/lesson_03_functions/calculator.py

```diff
@@ -0,0 +1,97 @@
+import sys
+
+
+def input_values():
+    print("Input 1st operand: ")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    return add_var_1, add_var_2
+
+
+def add():
+    print("ADDING")
+    add_var_1, add_var_2 = input_values()
+    print("Result:")
+    print(add_var_1 + add_var_2)
+
+
+def subtract():
+    print("SUBTRACT")
+    add_var_1, add_var_2 = input_values()
+    print("Result:")
+    print(add_var_1 - add_var_2)
+
+
+def multiply():
+    print("MULTIPLY")
+    add_var_1, add_var_2 = input_values()
+    print("Result:")
+    print(add_var_1 * add_var_2)
+
+
+def divide():
+    print("DIVIDE")
+    add_var_1, add_var_2 = input_values()
+    print("Result:")
+    print(add_var_1 / add_var_2)
+
+
+def power():
+    print("POWER")
+    add_var_1, add_var_2 = input_values()
+    print("Result:")
+    print(add_var_1 ** add_var_2)
+
+
+def help():
+    print("HELP")
+    print_options()
+
+
+def quit():
+    print("GOOD BYE")
+    sys.exit()
+
+
+def print_options():
+    print("a - add")
+    print("s - subtract")
+    print("m - multiply")
+    print("d - divide")
+    print("p - power")
+    print("h,? - help")
+    print("q - QUIT")
+
+
+CALCULATOR_DICT = {
```

Comment:
Move CALCULATOR_DICT at the top after imports, this lets you keep your module more ordered.